### PR TITLE
Minor typos

### PR DESCRIPTION
**Pull request by mikedotexe**
(Minor) Documentation content update for Authz, Fee Grant, Denoms.

This PR addresses a couple typos I saw while going through these amazing resources.

I also freaked out when saw some environment variables with `_KEY` in them like here:

https://github.com/cosmos/sdk-tutorials/blob/9ff84b94b7f5b4f49750afaefb87386a7931c94e/tutorials/understanding-feegrant/index.md?plain=1#L167

and realized that commands such as `simd keys show alice -a` are just showing the address, not the key. phew.

So I did a little search-and-replace to make `ALICE_KEY` and `BOB_KEY` into `ALICE_ADDRESS` and `BOB_ADDRESS` to add a little clarity.

Then the PR template had a typo and sniped it.

### Change scope

The changes in this Pull Request include (please tick all that apply):

- [X] Small language/grammar fixes
- [ ] Small content fixes 
- [ ] Addition of new content
- [ ] Sample code/command updates
- [ ] Platform fixes
- [ ] Other

### CODE REVIEW

File: tutorials/understanding-feegrant/index.md [hunk 1]
```diff
@@ -107,8 +107,8 @@ To avoid having to copy and paste the user addresses, now is a good time to expo
 :::
 
 ```sh
-$ export ALICE_KEY=$(simd keys show alice -a)
-$ export BOB_KEY=$(simd keys show bob -a)
+$ export ALICE_ADDRESS=$(simd keys show alice -a)
+$ export BOB_ADDRESS=$(simd keys show bob -a)
```
Comment (xavierlepretre) [inline tutorials/understanding-feegrant/index.md:111]:
```suggestion
$ export ALICE=$(simd keys show alice --address)
$ export BOB=$(simd keys show bob --address)
```

What do you think about just using `ALICE` and `BOB` if we use `--address` to make clear?

**Comment by mikedotexe** [comment]
Thanks for the suggestion there, @xavierlepretre!

File: tutorials/understanding-feegrant/index.md [hunk 2]
```diff
@@ -164,22 +164,22 @@ Before Bob can send `kudos` to Alice, you must set up an allowance for Bob so th
 The `BasicAllowance` is a permission for a grantee to use up fees until the `spend_limit` or `expiration` is reached. Open up a new terminal window and create an allowance with a spend limit of `100000stake` and no expiration date:
 
 ```sh
-$ simd tx feegrant grant $ALICE_KEY $BOB_KEY --from alice --spend-limit 100000stake
+$ simd tx feegrant grant $ALICE_ADDRESS $BOB_ADDRESS --from alice --spend-limit 100000stake
```
Comment (xavierlepretre) [inline tutorials/understanding-feegrant/index.md:167]:
```suggestion
$ simd tx feegrant grant $ALICE $BOB --from alice --spend-limit 100000stake
```

File: tutorials/understanding-feegrant/index.md [hunk 3]
```diff
@@ -164,22 +164,22 @@ Before Bob can send `kudos` to Alice, you must set up an allowance for Bob so th
...
 ```
 
 View the allowance:
 
 ```sh
-$ simd q feegrant grants $BOB_KEY
+$ simd q feegrant grants $BOB_ADDRESS
```
Comment (xavierlepretre) [inline tutorials/understanding-feegrant/index.md:173]:
```suggestion
$ simd q feegrant grants $BOB
```

File: tutorials/understanding-feegrant/index.md [hunk 4]
```diff
@@ -164,22 +164,22 @@ Before Bob can send `kudos` to Alice, you must set up an allowance for Bob so th
...
 
 ## Send Tokens
 
 First, let's check the balances of Alice and Bob. Verifying the initial balance provides a baseline so that you can later confirm if your transaction was successful:
 
 ```sh
-$ simd q bank balances $ALICE_KEY
-$ simd q bank balances $BOB_KEY
+$ simd q bank balances $ALICE_ADDRESS
+$ simd q bank balances $BOB_ADDRESS
```
Comment (xavierlepretre) [inline tutorials/understanding-feegrant/index.md:182]:
```suggestion
$ simd q bank balances $ALICE
$ simd q bank balances $BOB
```

File: tutorials/understanding-feegrant/index.md [hunk 5]
```diff
@@ -189,22 +189,22 @@ Any transaction that is sent using the `tx` command can use the `--fee-account`
 Send `kudos` tokens from Bob to Alice, while Alice pays the fees:
 
 ```sh
-$ simd tx bank send $BOB_KEY $ALICE_KEY 100kudos --from bob --fee-account $ALICE_KEY --fees 500stake
+$ simd tx bank send $BOB_ADDRESS $ALICE_ADDRESS 100kudos --from bob --fee-account $ALICE_ADDRESS --fees 500stake
```
Comment (xavierlepretre) [inline tutorials/understanding-feegrant/index.md:192]:
```suggestion
$ simd tx bank send $BOB $ALICE 100kudos --from bob --fee-account $ALICE --fees 500stake
```

File: tutorials/understanding-feegrant/index.md [hunk 6]
```diff
@@ -189,22 +189,22 @@ Any transaction that is sent using the `tx` command can use the `--fee-account`
...
 ```
 
 Look at the balances again:
 
 ```sh
-$ simd q bank balances $ALICE_KEY
-$ simd q bank balances $BOB_KEY
+$ simd q bank balances $ALICE_ADDRESS
+$ simd q bank balances $BOB_ADDRESS
```
Comment (xavierlepretre) [inline tutorials/understanding-feegrant/index.md:199]:
```suggestion
$ simd q bank balances $ALICE
$ simd q bank balances $BOB
```

File: tutorials/understanding-feegrant/index.md [hunk 7]
```diff
@@ -189,22 +189,22 @@ Any transaction that is sent using the `tx` command can use the `--fee-account`
...
 ```
 
 Notice how Alice has `500stake` less than before. The `500stake` was added to the transaction that Bob signed.
 
 View the allowance again:
 
 ```sh
-$ simd q feegrant grants $BOB_KEY
+$ simd q feegrant grants $BOB_ADDRESS
```
Comment (xavierlepretre) [inline tutorials/understanding-feegrant/index.md:207]:
```suggestion
$ simd q feegrant grants $BOB
```

File: tutorials/understanding-feegrant/index.md [hunk 8]
```diff
@@ -216,13 +216,13 @@ The granter can revoke the allowance from the grantee using the `revoke` command
 Revoke allowance:
 
 ```sh
-$ simd tx feegrant revoke $ALICE_KEY $BOB_KEY --from alice
+$ simd tx feegrant revoke $ALICE_ADDRESS $BOB_ADDRESS --from alice
```
Comment (xavierlepretre) [inline tutorials/understanding-feegrant/index.md:219]:
```suggestion
$ simd tx feegrant revoke $ALICE $BOB --from alice
```

File: tutorials/understanding-feegrant/index.md [hunk 9]
```diff
@@ -216,13 +216,13 @@ The granter can revoke the allowance from the grantee using the `revoke` command
...
 ```
 
 View the allowance:
 
 ```sh
-$ simd q feegrant grants $BOB_KEY
+$ simd q feegrant grants $BOB_ADDRESS
```
Comment (xavierlepretre) [inline tutorials/understanding-feegrant/index.md:225]:
```suggestion
$ simd q feegrant grants $BOB
```

**Review by xavierlepretre — CHANGES_REQUESTED** [review]
That would be where it is used too.

File: tutorials/understanding-feegrant/index.md [hunk 10]
```diff
@@ -164,22 +164,22 @@ Before Bob can send `kudos` to Alice, you must set up an allowance for Bob so th
...
 ```sh
-$ simd tx feegrant grant $ALICE_KEY $BOB_KEY --from alice --spend-limit 100000stake
+$ simd tx feegrant grant $ALICE $BOB --from alice --spend-limit 100000stake
 ```
 
 View the allowance:
 
 ```sh
-$ simd query feegrant grants $BOB_KEY
+$ simd q feegrant grants $BOB
```
Comment (xavierlepretre) [inline tutorials/understanding-feegrant/index.md:173]:
```suggestion
$ simd query feegrant grants $BOB
```
That is a correct way of making a query, but for a tutorial, we prefer to have the explicit long form, which makes for easier reading.
The experienced programmer will soon enough be looking for ways to optimise key strokes on their own.

**Review by xavierlepretre — CHANGES_REQUESTED** [review]

**Comment by coldice** [comment]
Hey @xavierlepretre, this has been left unattended for a bit - I think the OP's contribution is clear, I do like the simplification of `ALICE_ADDRESS` to just `ALICE`. I follow your comment regarding the verbosity of flags - I propose we adjust that on this PR, deconflict and then get this merged?

**Comment by xavierlepretre** [comment]
I still want him to unfix the `query` -> `q` he did.

**Comment by xavierlepretre** [comment]
Replaced by [PR 1241](https://github.com/cosmos/sdk-tutorials/pull/1241).

**Pull request closed without merging**